### PR TITLE
Adding duration unit in trivy timeout

### PR DESCRIPTION
**Pull request by navina**
Fixing an error in the previous PR (https://github.com/apache/pinot/pull/10134) 
Makes `pinot_vuln_check.yml` nightly

### CODE REVIEW

**Comment by navina** [comment]
@snleee  / @gortiz  / @xiangfu0 : are we ok with moving the trivy check to a nightly trigger, rather than on every commit and PR? We can add a badge in our repo to indicate the status of this job.

**Comment by snleee** [comment]
I'm fine with the change but we will need to make one time effort to clean up the test. I think that may involve in bumping up multiple libraries.

**Review by snleee — APPROVED** [review]

File: .github/workflows/pinot_vuln_check.yml [hunk 1]
```diff
@@ -19,18 +19,8 @@
...
-  pull_request:
-    branches:
-      - master
-    paths:
-      - "**/pom.xml"
-      - "**/package.json"
-      - "**/package-lock.json"
-      - "docker/images/pinot/**"
-      - ".github/workflows/**"
+  schedule:
```
Comment (walterddr) [inline .github/workflows/pinot_vuln_check.yml:22]: Why do we need a cron schedule? Let's still do triggered based?

Comment (navina) [reply .github/workflows/pinot_vuln_check.yml:22]: @walterddr trigger based on what ? can you tell me how?

**Review by xiangfu0 — APPROVED** [review]

**Comment by xiangfu0** [comment]
My only concern is that who will be responsible for the trivy check results and version bump.

File: .github/workflows/pinot_vuln_check.yml (hunk 1 above)
Comment (gortiz) [reply .github/workflows/pinot_vuln_check.yml:22]: We were talking about changing this check to do not run on each push but run periodically in order to do not mark as erroneous PRs that do not actually introduce the vulnerability. On the other hand, we would lose the ability to detect when a PR adds a known vulnerability. Instead we would need to be notified by this periodic cron. 

The big question here is how are we going to know when this cron fails.

**Comment by gortiz** [comment]
I understand that the initial idea was to increase the timeout but we are using this PR to change the way trivy is triggered. Therefore I would suggest to change the title of the PR

**Comment by navina** [comment]
> I understand that the initial idea was to increase the timeout but we are using this PR to change the way trivy is triggered. Therefore I would suggest to change the title of the PR

Yes. I will update the title, if needed before merging.

**Comment by Jackie-Jiang** [comment]
This change is included in #10150 which also fixes the trivy failure

**Pull request closed without merging**